crates/electrum/src/bdk_electrum_client.rs

```diff
@@ -22,6 +22,8 @@ pub struct BdkElectrumClient<E> {
     tx_cache: Mutex<HashMap<Txid, Arc<Transaction>>>,
     /// The header cache
     block_header_cache: Mutex<HashMap<u32, Header>>,
+    /// Cache of transaction anchors
+    anchor_cache: Mutex<HashMap<(Txid, BlockHash), ConfirmationBlockTime>>,
```

Comment:
Can be done in a follow up, but we can have a `populate_anchor_cache` method. This will speed things up even more for subsequent syncs.

Reply:
Ticket made: https://github.com/bitcoindevkit/bdk/issues/1982